data.json

```diff
@@ -718,5 +736,23 @@
     "image": "http://devblogs.microsoft.com/xamarin/wp-content/uploads/sites/44/2019/10/medium_25686-c9dd82896c9e66bb82600f256444abe6.png",
     "dateAdded": "2019-10-08T21:00:00.000Z",
     "tags": ["stickers", "hacktoberfest", "clothing", "expired"]
+  },
+  {
+    "name": "Xamarin by Microsoft",
+    "difficulty": "hard",
+    "description": "Make quality pull requests to issues that are 'approved item of work' t-shirt and other swag.",
```

Comment:
Can you cleanup a bit this description so it's clear what is required to get what swag? Thanks